### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Version 4.5.1
+===============
+- **ADAL 4.5.0 and lower fails to install due to a version conflict on Android projects targeting Xamarin.Android.Support libraries version 28.x**. The ADAL Xamarin.Android.Support libraries have been updated to 27.0.2.1, the maximum version on Android8.1. [ADAL issue #1486](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1486)
+- **ADAL occassionally threw a NullReferenceException during Http calls**. Now a relevant timeout AdalServiceException will be thrown. [ADAL issue #1489](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1489)
```
Comment (jmprieur) [inline changelog.txt]: ADAL occassionally threw a NullReferenceException during Http calls **when there were network timeouts**

**Review by jmprieur — APPROVED** [review]
LGTM
A small suggestion.
Thanks @jennyf19

File: changelog.txt (hunk 1 above)
Comment (jennyf19) [reply changelog.txt]: done. thank you.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,8 @@
+Version 4.5.1
+===============
+- **ADAL 4.5.0 and lower fails to install due to a version conflict on Android projects targeting Xamarin.Android.Support libraries version 28.x**. The ADAL Xamarin.Android.Support libraries have been updated to 27.0.2.1, the maximum version on Android8.1. [ADAL issue #1486](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1486)
+- **ADAL occassionally threw a NullReferenceException during Http calls when there were network timeouts**. Now a relevant timeout AdalServiceException will be thrown. [ADAL issue #1489](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/issues/1489)
```
Comment (henrik-me) [inline changelog.txt:4]: >occassionally [](start = 9, length = 13)

occasionally

Comment (jennyf19) [reply changelog.txt:4]: @henrik-me seriously? ugh...i'll fix it (nice catch)

**Pull request merged**